include/PushPlate.h

```diff
@@ -0,0 +1,54 @@
+#include "Definitions.h"
+#include "Access.h"
+
+struct PushPlate
+{
+    Bool active         = false; /* If being pressed by something. */
+    Bool activeprevious = false; /* Previous state of 'active'.    */
+    tid  tid            = 0;     /* Unique ID of this PushPlate.   */
+};
+
+struct PushPlateSignal
+{
+    tid  tid    = 0;     /* ID of the PushPlate that was triggered. */
```

Comment:
`tid` as a typename should be more explicit. Maybe `TileId` or `TypeId` or something like that?

Reply:
Given that `tid` is alias for `uint32` and in Definitions.h we already have `U32`, I would probably stick with that, but since I didn't write this code, I'll consult with team member and see what he thinks about changing typename. :)